ixmp4/data/db/run/repository.py

```diff
@@ -176,17 +174,3 @@ def unset_as_default_version(self, id: int) -> None:
 
         run.is_default = False
         self.session.commit()
-
-
-def select_joined_run_index(repository, **kwargs) -> db.sql.Select:
```

Comment:
For what it's worth, I added this function here so that I could use it later to streamline the following almost-duplicate code parts:

https://github.com/iiasa/ixmp4/blob/3f45bba201de4a448f5e7be930b9e6593fb59d49/ixmp4/core/run.py#L94

https://github.com/iiasa/ixmp4/blob/3f45bba201de4a448f5e7be930b9e6593fb59d49/ixmp4/data/db/iamc/datapoint/repository.py#L111

Reply:
Yeah i think id take a look at this again in some sort of selection overhaul PR. The problem is that depending on what table is selected the joins have to be different (and there was a bug in the code applying filters twice) so i would try to find some new abstraction for this and the filters to make it less error prone in the future.

Reply:
I also had to rename the columns selected ("model"->"model_name") because postgres 15 does not like columns in the result set with the same name as a table from the query